docs/iceberg-rest-service.md

```diff
@@ -138,6 +135,20 @@ For other Iceberg OSS properties not managed by Gravitino like `client.security-
 Please set the `gravitino.iceberg-rest.warehouse` parameter to `oss://{bucket_name}/${prefix_name}`. Additionally, download the [Aliyun OSS SDK](https://gosspublic.alicdn.com/sdks/java/aliyun_java_sdk_3.10.2.zip) and copy `aliyun-sdk-oss-3.10.2.jar`, `hamcrest-core-1.1.jar`, `jdom2-2.0.6.jar` in the classpath of Iceberg REST server, `iceberg-rest-server/libs` for the auxiliary server, `libs` for the standalone server.
 :::
 
+#### Other storages
+
+For storages that are not inherently integrated into Gravitino Iceberg REST service, you can manage them effectively through custom catalog properties.
+
+| Configuration item               | Description                                                                             | Default value | Required | Since Version |
+|----------------------------------|-----------------------------------------------------------------------------------------|---------------|----------|---------------|
+| `gravitino.iceberg-rest.io-impl` | The IO implementation for `FileIO` in Iceberg, please use the full qualified classname. | (none)        | No       | 0.6.0         |
+
+For other custom properties like `security-token` to pass to `FileIO`, you could config it directly by `gravitino.iceberg-rest.security-token`.
```

Comment:
Can you explain more here? For example, if user has a custom FileIO implementation called "A", then set configuration like "gravitino.iceberg-rest.xxx", how does this "A" know this configuration "xxx"?

Reply:
updated